js/components/controlBar.js

```diff
@@ -727,14 +696,44 @@ var ControlBar = createReactClass({
               null
           }
           height={this.props.skinConfig.controlBar.logo.height} />
+      ),
+
+      skipControls: (
+        <SkipControls
+          key={CONSTANTS.CONTROL_BAR_KEYS.SKIP_CONTROLS}
+          buttonConfig={Utils.getPropertyValue(
+            this.props.skinConfig,
+            'skipControls.buttonsWithPlayPause',
+            {}
+          )}
+          forceShowButtons={true}
+          maxWidth={200}
+          className={'oo-absolute-centered oo-control-bar-item'}
```

Comment:
We can probably change this later, but this could be just `className="
oo-absolute-centered oo-control-bar-item"`

Reply:
Added